src/VisualStudio/Core/Def/Commands.vsct

```diff
@@ -529,6 +542,14 @@
     <CommandPlacement guid="guidRoslynGrpId" id="cmdidRemoveUnusedReferences" priority="0xF200">
       <Parent guid="guidSHLMainMenu" id="IDG_VS_CTXT_REFROOT_ADD" />
     </CommandPlacement>
+
+    <CommandPlacement guid="guidRoslynGrpId" id="cmdidSyncNamespaces" priority="0xF200">
+      <Parent guid="guidSHLMainMenu" id="IDG_VS_CTXT_PROJECT_ADD" />
```

Comment:
Why on this group?

Reply:
@mikadumont Which grouping would we like this command to appear in?

Reply:
Oh I read this incorrectly, where in right-click context menu is this showing up? Can you right-click a folder within a project and choose Add -> and see if it shows up?

Reply:
It does not show up.